spring-cloud-gcp-examples/spring-cloud-gcp-sql-example/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.cloud.gcp.sql.database-name=[database-name]
+spring.cloud.gcp.sql.instance-connection-name=[instance-connection-name]
```

Comment:
Don't we need all other properties for `projectId`, `credentials` etc?

Reply:
Not if users are authenticated with the Cloud SDK. I added a bullet point in the README for that matter:

4. https://cloud.google.com/sdk/gcloud/reference/auth/login[Authenticate in the Cloud SDK] or add
a `spring.cloud.gcp.sql.credentials-location` property with the path on your local file system to
a credentials file, prepended with `file:`.